### PR TITLE
fix verificatio on ci

### PR DESCRIPTION
**Pull request by matklad**

### CODE REVIEW

**Comment by matklad** [comment]
bors r+
🤖

File: crates/tools/src/main.rs [hunk 1]
```diff
@@ -43,7 +43,7 @@ fn main() -> Result<()> {
     {
         "install-code" => install_code_extension()?,
         "gen-tests" => gen_tests(mode)?,
-        "gen-syntax" => generate(Overwrite)?,
```
Comment (flodiebold) [inline crates/tools/src/main.rs]: oh, but why did it fail locally then?

**Pull request merged**